micronaut/pom.xml

```diff
@@ -45,6 +45,11 @@
       <artifactId>micronaut-function-aws</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>io.micronaut.crac</groupId>
+      <artifactId>micronaut-crac</artifactId>
+      <version>1.2.0</version>
```

Comment:
Micronaut BOM v3.8.7 brings 1.1.2. If you really need 1.2.0, I'd recommend omitting the version here and setting the following property:

```xml
<micronaut.crac.version>1.2.0</micronaut.crac.version>
```

It should make future version upgrades easier.

Reply:
Good callout @alvarosanchez . I was looking for the latest version.

Reply:
I'll add @timyates and @yawkat to the loop as our CRaC and benchmarking experts, respectively :)